core/src/test/java/feign/template/QueryTemplateTest.java

```diff
@@ -19,6 +19,7 @@
 import feign.Util;
 import java.util.Arrays;
 import java.util.Collections;
+import javax.management.Query;
```

Comment:
I don't think you really meant to include this =)

Reply:
Right.  I'll remove it.